keyboards/centromere/keymaps/mini_bom/config.h

```diff
@@ -0,0 +1,12 @@
+ 	#pragma once
+	
+	#ifdef COMBO_ENABLE
+	#    define COMBO_COUNT 10
+	#    define COMBO_TERM 50
+	#endif
+
+    #ifdef TAPPING_TERM
+ 	#undef TAPPING_TERM
+	#endif
+ 	#define TAPPING_TERM 200
+    #define IGNORE_MOD_TAP_INTERRUPT
```

Comment:
```suggestion
/* Copyright 2020 keyboard-magpie
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 2 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

#pragma once

#ifdef COMBO_ENABLE
#    define COMBO_COUNT 10
#    define COMBO_TERM 50
#endif

#ifdef TAPPING_TERM
#    undef TAPPING_TERM
#endif
#define TAPPING_TERM 200
#define IGNORE_MOD_TAP_INTERRUPT
```